spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -387,7 +387,9 @@ def test_runcell(tmpdir):
 
         # Execute runcell
         client.execute(u"runcell('', r'{}')".format(to_text_string(p)))
-        client.get_shell_msg(block=True, timeout=TIMEOUT)
+        msg = client.get_shell_msg(block=True, timeout=TIMEOUT)
+        print(msg['content'])
+        assert "cell-tests.py" in msg['content']
```

Comment:
@ccordoba12 I can't get the output from the print statement in the code. Any suggestions? 

I can't use inspect because I remove the definition. If I don't remove it and then inspect I get the filename of the test.

Reply:
Instead of printing `__file__`, please assign the contents of `__file__` to a variable, then inspect its contents. That should be enough.

Reply:
Thank you! That works of course. I was too laser focused on trying to check `__file__` as a name rather than the content.

Reply:
Nice!